reana_client/cli/files.py

```diff
@@ -388,7 +388,68 @@ def delete_files(ctx, workflow, filenames, access_token):
             err=True)
 
 
+@click.command('mv')
+@click.argument('source')
+@click.argument('target')
+@click.option(
+    '-w',
+    '--workflow',
+    default=os.environ.get('REANA_WORKON', None),
+    help='Name or UUID of the workflow you are moving files for. '
+         'Overrides value of REANA_WORKON environment variable.')
+@add_access_token_options
+@click.pass_context
+def move_files(ctx, source, target, workflow, access_token):  # noqa: D301
+    """Move files within workspace.
+
+    Examples:\n
+    \t $ reana-client mv data/input.txt input/input.txt
+
+    """
+    logging.debug('command: {}'.format(ctx.command_path.replace(" ", ".")))
+    for p in ctx.params:
+        logging.debug('{param}: {value}'.format(param=p, value=ctx.params[p]))
+
+    if not access_token:
+        click.echo(
+            click.style(ERROR_MESSAGES['missing_access_token'], fg='red'),
+            err=True)
+        sys.exit(1)
+    if workflow:
+        try:
+            current_status = get_workflow_status(workflow,
+                                                 access_token).get('status')
+            if current_status == 'running':
+                click.echo(
+                    click.style('File(s) could not be moved for running '
+                                'workflow', fg='red'),
+                    err=True)
+                sys.exit(1)
+            files = list_files(workflow, access_token)
+            current_files = [file['name'] for file in files]
+            for item in current_files:
```

Comment:
I think this condition should be: `if source not in current_files`
or even just do the check directly in RWC to not have 2 api calls
```
(reana-3.6) ➜  reana-demo-root6-roofit git:(master) ✗ rc ls 
NAME             SIZE   LAST-MODIFIED      
code/fitdata.C   1648   2019-02-12T15:40:32
code/gendata.C   1937   2019-02-12T15:40:31
(reana-3.6) ➜  reana-demo-root6-roofit git:(master) ✗ rc mv code/fitdata.C code/fitdata2.C
Target file(s) code/fitdata2.C does not exist in workspace ['code/fitdata.C', 'code/gendata.C']
```

Reply:
if this is changed it works nicely :+1:

Reply:
I had condition `if source not in current_files` before but, then movement of dirs does not work.

Reply:
Yes, you are right, but then `mv` won't work for directories. I am thinking how to allow it.

Reply:
I added more logic, still not perfect, but allow movement of dirs.